webapp/components/test/test-wpt-amend-metadata.html

```diff
@@ -0,0 +1,59 @@
+<!doctype html>
```

Comment:
This file should be named simply `wpt-amend-metadata.html`, without the `test-` prefix (which is already implied by the directory name.)